src/SmoFacade/Server.cs

```diff
@@ -178,11 +178,11 @@ public Database Database(string dbName)
     ///   wait
     /// </param>
     /// <param name="fileRelocation">Option for renaming files during the restore.</param>
-    public void Restore(IEnumerable<BackupMetadata> backupOrder, string databaseName,
+    public void Restore(IEnumerable<StripedBackupSet> stripedBackupSetChain, string databaseName,
       Func<int, TimeSpan> retryDurationProvider,
       Func<string, string> fileRelocation = null)
     {
-      var policy = Policy
+      var retryPolicy = Policy

```

Comment:
this can probably be better served as a class static variable so it doesn't have to be passed around as a parameter

Reply:
The `restore()` function takes in a `Func<int, TimeSpan> retryDurationProvider` parameter and we use that to create this policy.

I'm wondering if we can pass it in the constructor of the `Server` class instead and then make `retryPolicy` a member variable?
Or maybe just have a static retryDuration?

Reply:
ah.  forgot that it depends on the parameter. Leave it as is